client/src/components/VersionSaveComponent.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Input } from 'antd'
+import React, { useState } from 'react'
+import { GetAnswerDocument, useSaveVersionMutation } from '../generated/graphql'
+import { messageService } from '../services/message'
+
+const VersionSaveComponent: React.FC<{
+  answerId: string
+  commitMessage: string
+  versionId: string
+}> = props => {
+  const [inputState, setInputState] = useState('')
+
+  const [saveVersion] = useSaveVersionMutation({
+    onCompleted: ({ saveVersion }) => {
+      if (saveVersion === true) {
+        messageService.success('New version saved')
+      } else {
+        messageService.error('No changes detected, not saving')
+      }
+    },
+    variables: {
+      id: props.answerId,
+      commitMessage: inputState
+    },
+    refetchQueries: [
+      {
+        query: GetAnswerDocument,
+        variables: {
+          id: props.answerId
+        }
+      }
+    ]
+  })
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'row' }}>
+      <Input
+        placeholder="Commit Message"
+        value={inputState}
+        style={{ height: '32px', minHeight: '26px', marginRight: '10px' }}
+        onChange={e => {
+          setInputState(e.target.value)
+        }}
+      />
+
+      <Button
+        type="primary"
+        onClick={() => {
+          saveVersion({
+            variables: { id: props.answerId, commitMessage: inputState }
+          })
+          setInputState('')
+        }}
+      >
+        Save version
+      </Button>
+    </div>
```

Comment:
Please use the ui framework (`antd`)

```suggestion
    <Row gutter={8}>
      <Col>
        <Input
          placeholder="Commit Message"
          value={inputState}
          style={{ height: '32px', minHeight: '26px' }}
          onChange={e => {
            setInputState(e.target.value)
          }}
        />
      </Col>
      <Col>
        <Button
          type="primary"
          onClick={() => {
            saveVersion({
              variables: { id: props.answerId, commitMessage: inputState }
            })
            setInputState('')
          }}
        >
          Save version
        </Button>
      </Col>
    </Row>
```